tests/evergreen/test_util.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+import evergreen.util as under_test
+
+SAMPLE_DATE_STRING = "2019-02-13T14:55:37.000Z"
+
+
+class TestParseEvergreenDateTime:
```

Comment:
shouldn't these be new style classes?

Reply:
It doesn't really really matter. The class is just used to hold a collection of tests. I can switch it.

Reply:
Up to you really. I thought there were python3 implications for old-style classes.